simulation/traffic_simulator/src/hdmap_utils/hdmap_utils.cpp

```diff
@@ -100,7 +144,7 @@ auto HdMapUtils::canonicalizeLaneletPose(
       canonicalized.lanelet_id = ids[0];
     }
   }
-  while (canonicalized.s > getLaneletLength(canonicalized.lanelet_id)) {
+  while (canonicalized.s > (getLaneletLength(canonicalized.lanelet_id) + 0.2)) {
```

Comment:
/// @note Hard coded parameter! (Please describe why the parameter is 0.2) and leave a comment explaining what this parameter does and why it should be that value.

Reply:
I've removed this hardcoded value because it didn't work for all cases.
I've added checking if a pose can be canonicalized in `PedestrianActionNode::estimateLaneletPose()` function
This change is in https://github.com/tier4/scenario_simulator_v2/pull/992/commits/1bba583ce23693331a4105b77480063bd325ec11